lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -96,10 +147,20 @@ mkJormungandrLayer mgr baseUrl = JormungandrLayer
         Left e -> do
             throwM (ErrUnexpectedNetworkFailure ctx e)
 
-    cGetTipId = client api
+    cGetTipId
+        :<|> cGetBlock
+        :<|> cGetBlockDescendantIds
+        = client api
 
 data ErrUnexpectedNetworkFailure
     = ErrUnexpectedNetworkFailure Link ServantError
     deriving (Show)
 
 instance Exception ErrUnexpectedNetworkFailure
+
+data ErrGetDescendants
+    = ErrGetDescendantsNetworkUnreachable ErrNetworkUnreachable
+    | ErrGetDescendantsParentNotFound
```

Comment:
Think about giving more _details_ and context to errors. Here for instance, it would be nice to know at least which parent isn't found.